ui/app/components/generate-credentials-totp.hbs

```diff
@@ -30,7 +30,7 @@
     {{#if @totpCodePeriod}}
       <label class="code-validity">
```

Comment:
since this custom class is just adding padding-left can we use a general helper here? `has-left-padding-s`